### ISSUE TITLE
The use of SnapKit in a framework is causing a crash

### ISSUE THREAD

**Issue by fcusid230111** [Masonry/Snap#775]
### New Issue Checklist

🚫 If this template is not filled out your issue **will** be closed with no comment. 🚫

* [T] I have looked at the [Documentation](http://snapkit.io/docs)
* [T] I have read the [F.A.Q.](http://snapkit.io/faq)
* [T] I have filled out this issue template.

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform                | iOS
 Platform Version        | iOS 16.2
 SnapKit Version         | 5.6.0
 Integration Method      | cocoapods
 

### Issue Description

The use of SnapKit in a framework is causing a crash. Please help me fix it.

<img width="749" alt="image" src="https://github.com/SnapKit/SnapKit/assets/122441558/912dcbc1-9918-449e-8b4c-61e21f3b392f">

<img width="1032" alt="image" src="https://github.com/SnapKit/SnapKit/assets/122441558/25ece97e-4d65-493a-abae-de83a6e0e150">

**Comment by zhongdaima** [comment]
Yes, me too. When using it in the framework, I found that it crashes when using make.size and make.center.equalToSuperview().

**Comment by bilibili11** [comment]
How do I resolve this constraint situation where the constraint associated with the centerY also crashes if it is applied to the same view

**Comment by robertjpayne** [comment]
Is `centerRedView` part of the view hierarchy? They will need a common ancestory for a superview constraint to work
